lib/node_modules/@stdlib/math/base/tools/hermitepoly/docs/img/equation_normalized_hermite_polynomials.svg

```diff
@@ -0,0 +1,79 @@
+<svg xmlns:xlink="http://www.w3.org/1999/xlink" width="29.988ex" height="5.676ex" style="vertical-align: -2.005ex;" viewBox="0 -1580.7 12911.7 2443.8" role="img" focusable="false" xmlns="http://www.w3.org/2000/svg" aria-labelledby="MathJax-SVG-1-Title">
```

Comment:
When you regenerate the SVG equations (update the label in the README), this file will need to be manually deleted.